src/stcal/jump/twopoint_difference.py

```diff
@@ -251,8 +251,8 @@ def find_crs(dataa, group_dq, read_noise, normal_rej_thresh,
                                     np.bitwise_or(gdq[integ, group, row, col - 1], jump_flag)
 
                     if cr_col[j] != ncols - 1:
-                        if (gdq[integ, group, row, col - 1] & sat_flag) == 0:
-                            if (gdq[integ, group, row, col - 1] & dnu_flag) == 0:
+                        if (gdq[integ, group, row, col + 1] & sat_flag) == 0:
```

Comment:
I think what actually needs to be done here is this:

```
                    if cr_col[j] != ncols - 1:
                        if (gdq[integ, group, row, col - 1] & sat_flag) == 0:
                            if (gdq[integ, group, row, col - 1] & dnu_flag) == 0:
                                gdq[integ, group, row, col - 1] =\
                                    np.bitwise_or(gdq[integ, group, row, col - 1], jump_flag)
                        if (gdq[integ, group, row, col + 1] & sat_flag) == 0:
                            if (gdq[integ, group, row, col + 1] & dnu_flag) == 0:
                                gdq[integ, group, row, col + 1] =\
                                    np.bitwise_or(gdq[integ, group, row, col + 1], jump_flag)
```

Reply:
this block runs if its not a column at the edge, so it should be checking col-1 and col+1 individually.

we also need to apply this logic to the rows above and below.

Reply:
It looks like to me your proposed version only checks to make sure this pixel is not in the last column. That applies to the first setting of the jump flag. The second line that sets the pixel to the left of the current pixel needs to have a check that this is not the first column. You seem to have removed that check.
All that I changed was correcting the typo in lines 254 and 255 changing the "-" to a "+". I worried that we're not looking at the same version of the code. Is that what you see?

Reply:
(summarizing our call)

the block above `if cr_col[j] != 0:` takes care of the left adjacent pixel, so this sign change takes care of the right. everything looks good here. apologies for my misunderstanding